src/domain/accounts/services/src/services/account_service_impl.rs

```diff
@@ -40,20 +40,16 @@ impl AccountServiceImpl {
 
 #[async_trait::async_trait]
 impl AccountService for AccountServiceImpl {
-    async fn account_by_id(
+    async fn get_account_by_id(
```

Comment:
Renamed here, rename in multi-version for consistency as well.

Reply:
Done